README.md

```diff
@@ -1,5 +1,5 @@
-buildah - a tool for building OCI images
-========================================
+buildah - a tool which facilites building OCI container images and containers
```

Comment:
I think this is confusing, since "containers" to most people mean the processes running inside of a container, IE something started by runc.
Buildah ultimate output is Container images, either OCI or upstream docker images.  These objects when in preparation state are referred to as "containers" or container images.  After they are committed they become OCI IMages or upstream docker images.